Formula/disque.rb

```diff
@@ -0,0 +1,17 @@
+class Disque < Formula
+  desc "Disque, an in-memory, distributed job queue"
+  homepage "https://github.com/antirez/disque"
+  url "https://github.com/antirez/disque/archive/1.0-rc1.tar.gz"
+  version "1.0-rc1"
```

Comment:
Well that's a problem. There's isn't one yet.

On Wed, Apr 20, 2016 at 3:07 PM Mike McQuaid <notifications@github.com>
wrote:

> In Formula/disque.rb
> <https://github.com/Homebrew/homebrew-core/pull/481#discussion_r60402946>:
>
> > @@ -0,0 +1,17 @@
> > +class Disque < Formula
> > +  desc "Disque, an in-memory, distributed job queue"
> > +  homepage "https://github.com/antirez/disque"
> > +  url "https://github.com/antirez/disque/archive/1.0-rc1.tar.gz"
> > +  version "1.0-rc1"
>
> This needs to be a stable (i.e. non-rc) version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/481/files/baddfc29ea24bd8e6bf740e7d1f6e16752f41111#r60402946>
>